dessia_common/workflow/core.py

```diff
@@ -1799,9 +1800,13 @@ def __init__(self, workflow: Workflow, input_values, output_value, values,
             end_time = time.time()
         self.end_time = end_time
         self.execution_time = end_time - start_time
-        self.variable_values = {workflow.variable_indices(k): v for k, v in values.items() if k.memorize}
-        WorkflowState.__init__(self, workflow=workflow, input_values=input_values, activated_items=activated_items,
-                               values=values, start_time=start_time, output_value=output_value, log=log, name=name)
+        self.variable_values = {workflow.variable_indices(k): v for k, v in values.items()
```

Comment:
Is it enough to just filter ? 
I mean, do we want to do something about non-serializable values ?